lib/acts_as_tree.rb

```diff
@@ -63,6 +65,7 @@ module ClassMethods
     #                            a custom column by passing a symbol or string.
     def acts_as_tree(options = {})
       configuration = {
+        primary_key:   "id",
```

Comment:
Could you change the default to the tables primary key? It is returned by calling the method `primary_key`.

That way it will work fine, if the model uses a non-standard primary key name.